src/components/Bag.jsx

```diff
@@ -0,0 +1,131 @@
+import { useDispatch, useSelector } from "react-redux";
+import { Flex, Popover, Text, Button } from "@sparrowengg/twigs-react";
+import { removeFromCart, setTotalAmount } from "../redux/actions";
+import { useNavigate } from "react-router-dom";
+import { useEffect, useMemo } from "react";
+const Bag = () => {
+  const cart = useSelector((state) => state.cart);
+  const totalAmount = useSelector((state) => state.totalAmount);
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    let tot = 0;
```

Comment:
@arun-121 can you give meaningful names for the variables

Reply:
changed from tot to total